### DISCUSSION TITLE
Release schedule

### DISCUSSION THREAD

**Discussion by davetapley** [msiemens/tinydb#500] [General]
I see it's coming up on a year since [last release](https://github.com/msiemens/tinydb/releases/tag/v4.7.0), any plans for a new one?

https://github.com/msiemens/tinydb/compare/6ef791efef598a0a161be8d86efa94f9ce056c5c...master

**Comment by msiemens** [comment]
That's a good point. So far, my strategy has been to delay putting out a new release for two or three weeks after a Pull Request has been merged in case another change gets accepted that could be included in the same release. So far, this resulted in 3-5 releases per year as there isn't *that* much active development going on. It seems like I forgot to set a reminder for myself after the commits to TinyDB in last July and November.

Maybe, a better strategy would be to have a recurring release date (e.g. monthly) and just skip releases that wouldn't contain any changes. But that would introduce more delay for changes that just didn't make it in time. And also I don't know if I can guarantee that I'll be able to make time on a fixed day in order to prepare a new release.

At any rate, I'll put together a new release in a couple of minutes.

**Reply by vchendrix** [reply]
Hey there.  Love TinyDB but I see that there hasn't been any activity in 2024.  Are there any future plans?

**Reply by msiemens** [reply]
You're right, there has been basically no development for the last year. As for future plans, I consider TinyDB to be in a sort of maintenance mode where it has all the features I think it needs and will try to fix major bugs, but other than that I'd rely on the community for contributions.

That said, for the past year I haven't really taken the time to sort through the PRs I _did_ get. It's just that the longer the backlog gets, the stronger the desire to procrastinate becomes 🙈 Still, I think it's a fair question to ask! I'll try to sort through the backlog over the next couple of weeks and also post an update on the project's status on the discussion tab and link to it from the README.

**Reply by msiemens** [reply]
I've posted a more extended project status update in the discussion forum: https://github.com/msiemens/tinydb/discussions/572
